wordle-web/src/scripts/wordsService.ts

```diff
@@ -7,9 +7,19 @@ export abstract class WordsService {
     return this.#words.includes(word)
   }
 
-  static validWords(): Array<string> {
+  static validWords(currentWord: string): Array<string> {
     //Todo
-    return new Array<string>()
+
+    const validWordList = new Array<string>()
+
+    for (const word of this.#words) {
+      if (word.startsWith(currentWord) && validWordList.length < 100) {
+        validWordList.push(word)
+      }
+    }
+
+    validWordList.sort()
+    return validWordList
```

Comment:
This is more of a comment but, this right here does restrict your wordlist from being only 100 different words. I do understand the necessity of doing this though. It does get very taxing if you display too many; I ended up making it so they had to at least type one letter which significantly cuts down the list but it is still quite the amount of words to display. I think it's fine but I thought I would at least throw it out there.